include/Options/Business/Redirect_Lang.php

```diff
@@ -30,6 +30,26 @@ public static function key(): string {
 		return 'redirect_lang';
 	}
 
+	/**
+	 * Adds information to the site health info array.
+	 *
+	 * @since 3.8
+	 *
+	 * @param array   $info    The current site health information.
+	 * @param Options $options An instance of the Options class providing additional configuration.
+	 *
+	 * @return array The updated site health information.
+	 */
+	public function add_to_site_health_info( array $info, Options $options ): array {
+		if ( $options->get( self::key() ) ) {
```

Comment:
```suggestion
		if ( $this->get() ) {
```
Let's use the current instance.